cumulusci/cli/tests/test_cci.py

```diff
@@ -1167,90 +1335,14 @@ def test_task_run_debug_after(self, set_trace):
                 no_prompt=True,
             )
 
-    def test_task_run_usage_error(self):
-        config = mock.Mock()
-        config.get_org.return_value = (None, None)
-        config.project_config = BaseProjectConfig(
-            None,
-            config={
-                "tasks": {
-                    "test": {"class_path": "cumulusci.cli.tests.test_cci.DummyTask"}
-                }
-            },
-        )
-        DummyTask._run_task.side_effect = TaskOptionsError
-
-        with self.assertRaises(click.UsageError):
```

Comment:
🤔: Presumably, these were removed because we're catching these globally, but are we certain that we don't need any of them to confirm that malformed input results in an exception?

Reply:
Yeah, I removed some of these tests because they were for exception handling codepaths that got replaced by the centralized handling. Testing that task classes validate their options properly belongs in the tests for the task class. I did keep test_task_run_invalid_option which tests the extra option validation that is done by task_run.